api/sdmx-rest.yaml

```diff
@@ -357,7 +359,7 @@ components:
             metadataproviderscheme,
             reportingtaxonomy,
             metadataprovisionagreement
```

Comment:
Is a comma missing after `metadataprovisionagreement`?

Reply:
Indeed, thanks @stratosn. Fixed in https://github.com/sdmx-twg/sdmx-rest/pull/185/commits/1a682f490096da1b9c59dd3f4cca15e13ad3b1ed